### PR TITLE
Add Celo Cannoli Testnet

### PR DESCRIPTION
**Pull request by aaronmgdr**
## Description

Adds config for the upcoming Celo Testnet "Cannoli"

## Additional Information

- [X ] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address: 0xad5DC090223b3f6b9a47C354B808FF593FfA1e25

### CODE REVIEW

File: packages/chains/src/celoCannoli.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  blockExplorers: {
+    default: {
+      name: 'Celo Explorer',
+      url: 'https://explorer.celo.org/cannoli',
+    },
+  },
+  contracts: {
+    multicall3: {
+      address: '0x1b5c5e5e7f1c1c1b5c5e5e7f1c1c1b5c5e5e7f1c',
```
Comment (jxom) [inline packages/chains/src/celoCannoli.ts:28]: I can't seem to find this address on the block explorer – can you point me to it?

Comment (aaronmgdr) [reply packages/chains/src/celoCannoli.ts]: fixed now

Comment (aaronmgdr) [reply packages/chains/src/celoCannoli.ts]: @jxom let me know if you would prefer if I removed the multicall part until the contract can be verified

File: packages/chains/src/celoCannoli.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  blockExplorers: {
+    default: {
+      name: 'Celo Explorer',
+      url: 'https://explorer.celo.org/cannoli',
+    },
+  },
+  contracts: {
+    multicall3: {
+      address: '0x5Acb0aa8BF4E8Ff0d882Ee187140713C12BF9718',
```
Comment (tmm) [inline packages/chains/src/celoCannoli.ts:28]: Would be great if you could verify this contract or deploy a new one so folks can make sure it's the multicall3 contract.

Comment (aaronmgdr) [reply packages/chains/src/celoCannoli.ts:28]: Contract has now been verified https://explorer.celo.org/cannoli/address/0x5Acb0aa8BF4E8Ff0d882Ee187140713C12BF9718/contracts#address-tabs

File: packages/chains/src/celoCannoli.ts (hunk 1 above)
Comment (aaronmgdr) [reply packages/chains/src/celoCannoli.ts]: now verified https://explorer.celo.org/cannoli/address/0x5Acb0aa8BF4E8Ff0d882Ee187140713C12BF9718/contracts#address-tabs

**Pull request merged**